src/ipc-handlers.ts

```
@@ -1267,6 +1268,15 @@ export async function isCATrusted(): Promise< boolean > {
 	return isRootCATrusted();
 }
 
+export async function trustCertificate(): Promise< void > {
+	const platform = process.platform;
+	if ( platform === 'win32' ) {
+		return trustRootCA();
```

I think we should consider returning a boolean from `trustRootCA` to indicate if the operation was successful. If it fails, we can display an error modal. Silently failing would not be very helpful for users.

---

That's a very good suggestion. I have addressed it.

---

I am thinking - why do we go with returning boolean, instead of typical approach to throw error from `trustRootCA`? It looks redundant.
I noticed that in PHP world (at least in dotcom) we have these approaches, but I have never seen that it was used in JS world, maybe because JS has good errors ecosystem and async nature. If we look at `handleTrustCertificate` - the handling with `! result` looks weird.
Would be cool to not mute errors inside `trustRootCA`, always throw errors outside and consumers should decide what to do with errors - mute / show dialog / etc.